packages/lesswrong/components/review/ReviewVotingPage2019.tsx

```diff
@@ -0,0 +1,617 @@
+import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
This page is not working. I think maybe @Raemon you decided that was ok with you? But if not then I think this is the first thing I've seen that would hold up shipping.